src/esp/assets/GenericInstanceMeshData.h

```diff
@@ -18,25 +17,12 @@
 namespace esp {
 namespace assets {
 
-/**
-@brief Pack primitive IDs into a texture
-
-1D texture won't be large enough so the data have to be put into a 2D texture.
-For simplicity on both the C++ and shader side the texture has a fixed width
-and height is dynamic, and addressing is done as
-`(gl_PrimitiveID % width, gl_PrimitiveID / width)`.
-*/
-Magnum::GL::Texture2D createInstanceTexture(
-    Corrade::Containers::ArrayView<uint32_t> objectIds);
-
 class GenericInstanceMeshData : public BaseMesh {
  public:
   struct RenderingBuffer {
     Magnum::GL::Mesh mesh;
     Magnum::GL::Buffer vbo;
```

Comment:
Maybe rename this to be something like `interleavedVertColorIdBuffer`.  I was confused when I saw `Magnum::GL::Buffer cbo;` deleted in the diff.

Reply:
With my suggestion to [std::move() the buffers into the mesh](https://github.com/facebookresearch/habitat-sim/pull/191#discussion_r321323044), this would become just a `GL::Mesh` in the end.